reset_user_access_right/README.rst

```diff
@@ -0,0 +1,45 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=======================
+Reset User Access Right
+=======================
+
+This module adds a button on users form view to reset groups access right.
```

Comment:
I'm sorry. I don't get what this is supposed to do (but haven't looked at the code yet).